src/main/java/uk/gov/hmcts/cmc/claimstore/services/CountyCourtJudgmentService.java

```diff
@@ -66,11 +63,11 @@ public Claim save(
 
         authorisationService.assertIsSubmitterOnClaim(claim, userDetails.getId());
 
-        countyCourtJudgmentRule.assertCountyCourtJudgementCanBeRequested(claim, countyCourtJudgment.getCcjType());
+        countyCourtJudgmentRule.assertCountyCourtJudgmentCanBeRequested(claim, countyCourtJudgment.getCcjType());
 
-        Optional<Response> response = claim.getResponse();
-        if (claim.getState().equals(ClaimState.STAYED)
-            && response.filter(ResponseUtils::isAdmissionResponse).isPresent()) {
+        if (claim.getState() == ClaimState.STAYED
```

Comment:
Why change? Its generally better to use equals() where the LHS could be null.

Reply:
```
definitelyPresent.equals(possiblyNull)
```
is preferred over
```
possiblyNull.equals(definitelyPresent)
```
but with `enum`s, we can just use `==`, which is null-safe. Remember: enums are explicitly singletons.